server/models/loan_model.js

```diff
@@ -0,0 +1,42 @@
+
+
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const loanModelSchema = new Schema(
+    {
+        userAddress: { type: String, required: true },
```

Comment:
Here we need to add the loan manager address and loan manager version

Reply:
server code is removed